syncd/Makefile.am

```diff
@@ -33,6 +33,12 @@ syncd_CPPFLAGS = $(DBGFLAGS) $(AM_CPPFLAGS) $(CFLAGS_COMMON) \
 				 -I/usr/include/sai $(SAIFLAGS)
 syncd_LDADD = -lhiredis -lswsscommon -lsai -lpthread
 
+if SAITHRIFT
+SAI_RPC_INCLUDE_DIR = ../libsaiserver/include
+SAI_RPC_LIB_DIR = ../libsaiserver/lib
+syncd_CPPFLAGS += -I$(SAI_RPC_INCLUDE_DIR) -DSAITHRIFT=yes
+syncd_LDADD += -L$(SAI_RPC_LIB_DIR) -lrpcserver -lthrift
```

Comment:
>-lthrift [](start = 48, length = 8)

do we need thrift explicitly? I thought rpcserver already has a dependency on that.